rllib/evaluation/rollout_worker.py

```diff
@@ -1567,7 +1567,13 @@ def set_state(self, objs: bytes) -> None:
                         config=policy_spec.config,
                     )
             if pid in self.policy_map:
-                self.policy_map[pid].set_state(state)
+                self.policy_map[pid].set_state(policy_state)
+
+        # Also restore mapping fn and which policies to train.
```

Comment:
These two functions must be part of a Worker's state, otherwise, it wouldn't know how to map in case policies change (added/removed).